src/tests/build.proj

```diff
@@ -529,6 +529,8 @@
       <CrossgenDir Condition="'$(TargetArchitecture)' == 'arm'">$(CrossgenDir)\x64</CrossgenDir>
       <CrossgenDir Condition="'$(TargetArchitecture)' == 'arm64'">$(CrossgenDir)\x64</CrossgenDir>
       <CrossgenDir Condition="'$(TargetArchitecture)' == 'x86'">$(CrossgenDir)\x64</CrossgenDir>
+      <Crossgen2Path>$(CrossgenDir)\crossgen2\crossgen2</Crossgen2Path>
+      <Crossgen2Path Condition="'$(TargetOS)' == 'windows'">$(Crossgen2Path).exe</Crossgen2Path>
```

Comment:
I think this is wrong, test framework gets crossgenned on the AzDO build machine so we should be checking <code>HostOS</code>, not <code>TargetOS</code> (not sure if that is actually causing any of the lab failures but I think it's still a bug).

Reply:
We don't support Windows<->Unix cross OS compilation, so it's debatable..